### PR TITLE
Go module support

### PR DESCRIPTION
**Pull request by s1061123**

### CODE REVIEW

File: cmd/example/main.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/clientcmd"
 
-	clientset "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/client/clientset/versioned"
+	clientset "github.com/K8sNetworkPlumbingWG/network-attachment-definition-client/pkg/client/clientset/versioned"
```
Comment (phoracek) [inline cmd/example/main.go]: What does this change imply to `dep` users? We faced an issue with `dep` being unable to handle upper-case letters https://github.com/K8sNetworkPlumbingWG/network-attachment-definition-client#networkattachmentdefinition-crd-client.

Just making sure this change won't break existing users.

Comment (s1061123) [reply cmd/example/main.go]: Currently this PR could be succeed to build command with go-module.
The build may be fail if you use non go-module build environment. Golang now makes go module default, hence it should be okey.

In the future, we should consider the change of github group name (from K8sNetworkPlumbingWG, mixed-case to others).

Comment (s1061123) [reply cmd/example/main.go]: As far as I looks dep, https://github.com/golang/dep/pull/1079 may fix the issue, I guess.

And there's some github projects that fix same stuff: 
https://github.com/hashicorp/consul-template/pull/1139/files

Comment (phoracek) [reply cmd/example/main.go]: I see. Fair enough. Thanks Tomo.

**Pull request merged**